src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -83,6 +83,10 @@
             if (ret = allHeaders.match(/^x-debug-token-link:\s+(.*)$/im)) {
                 stackElement.profilerUrl = ret[1];
             }
+            if (ret = allHeaders.match(/^x-debug-toolbar-replace:\s+(.*)$/im)) {
+                stackElement.toolbarReplaceFinished = false;
+                stackElement.toolbarReplace = ret[1];
```

Comment:
you treat this as a boolean later, but the match here is a string. So this indeed weird regarding the expected value of this header. It requires some explanation